packages/react/src/components/Menu/Menu.tsx

```diff
@@ -148,7 +148,11 @@ class Menu extends AutoControlledComponent<ReactProps<MenuProps>, MenuState> {
     },
   })
 
-  renderItems = (styles: ComponentSlotStylesPrepared, variables: ComponentVariablesObject) => {
+  renderItems = (
+    styles: ComponentSlotStylesPrepared,
+    variables: ComponentVariablesObject,
+    accessibility: Accessibility,
```

Comment:
It should be `AccessibilityBehavior`:

```suggestion
    accessibility: AccessibilityBehavior,
```